src/appengine/libs/issue_management/jira/issue_tracker_manager.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Issue tracker manager functions."""
+from __future__ import absolute_import
+
+from builtins import object
+import jira
+import json
+
+from config import db_config
+
+
+class IssueTrackerManager(object):
+  """Issue tracker manager."""
+
+  def __init__(self, project_name):
+    """"Construct an issue tracker manager instance based on parameters."""
+    self._client = None
+    self.project_name = project_name
+
+  @property
+  def client(self):
+    """HTTP Client."""
+    if self._client is None:
+      self._client = self._create_client()
+
+    return self._client
+
+  def _create_client(self):
+    """Return a client object for querying the issue tracker."""
+    config = db_config.get()
+    credentials = json.loads(config.jira_credentials)
+    jira_url = config.jira_url
+    jira_client = jira.JIRA(
+        jira_url, auth=(credentials['username'], credentials['password']))
+    return jira_client
+
+  def save(self, issue):
+    """Save an issue."""
+    self._update(issue)
+
+  def create(self):
+    """Create an issue."""
+    jira_project = self.project_name
```

Comment:
nit: no need for the jira_project var. Just use self.project_name directly below.

Reply:
ACK